### ISSUE TITLE
[JENKINS-75253] <html> element must have a lang attribute

### ISSUE THREAD

**Issue by lemeurherve** [lemeurherve-org/demo#811]
<p>We are using Jenkins for our GUI.  Our accessibility guidelines state the following:</p>

<p> The <a href="https://www.w3.org/TR/WCAG/#dfn-human-language-s" class="external-link" target="_blank" rel="nofollow noopener">human language</a> of each passage or phrase in the content should be <a href="https://www.w3.org/TR/WCAG/#dfn-programmatically-determinable" class="external-link" target="_blank" rel="nofollow noopener">programmatically determined</a> except for proper names, technical terms, words of indeterminate language, and words or phrases that have become part of the vernacular of the immediately surrounding text.</p>

<p>The default <a href="https://www.w3.org/TR/WCAG/#dfn-human-language-s" class="external-link" target="_blank" rel="nofollow noopener">human language</a> of each <a href="https://www.w3.org/TR/WCAG/#dfn-web-page-s" class="external-link" target="_blank" rel="nofollow noopener">Web page</a> or document or software can be <a href="https://www.w3.org/TR/WCAG/#dfn-programmatically-determinable" class="external-link" target="_blank" rel="nofollow noopener">programmatically determined</a>.</p>

<p>The fix for this issue found is to add "lang" attribute to <html> tag which will specify the default language of the page. Kindly add the "lang" attribute to <html> tag.</p>

<p> </p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=geetika_agarwal">geetika_agarwal</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-75253" target="_blank"><html> element must have a lang attribute</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We are using Jenkins for our GUI.  Our accessibility guidelines state the following:</p>

<p> The <a href="https://www.w3.org/TR/WCAG/#dfn-human-language-s" class="external-link" target="_blank" rel="nofollow noopener">human language</a> of each passage or phrase in the content should be <a href="https://www.w3.org/TR/WCAG/#dfn-programmatically-determinable" class="external-link" target="_blank" rel="nofollow noopener">programmatically determined</a> except for proper names, technical terms, words of indeterminate language, and words or phrases that have become part of the vernacular of the immediately surrounding text.</p>

<p>The default <a href="https://www.w3.org/TR/WCAG/#dfn-human-language-s" class="external-link" target="_blank" rel="nofollow noopener">human language</a> of each <a href="https://www.w3.org/TR/WCAG/#dfn-web-page-s" class="external-link" target="_blank" rel="nofollow noopener">Web page</a> or document or software can be <a href="https://www.w3.org/TR/WCAG/#dfn-programmatically-determinable" class="external-link" target="_blank" rel="nofollow noopener">programmatically determined</a>.</p>

<p>The fix for this issue found is to add "lang" attribute to &lt;html&gt; tag which will specify the default language of the page. Kindly add the "lang" attribute to &lt;html&gt; tag.</p>

<p> </p></pre>
</details>
</details>
